### ISSUE TITLE
[Feature Request] Display Multiple Time Zones Clocks

### ISSUE THREAD

**Issue by HectorB-2020** [mate-desktop/mate-panel#1307]
#### Expected behaviour
When you hover mouse pointer (cursor) above the system clock, it should display the current time for the home region and other configured locations.

#### Actual behaviour
"Click to view month calendar" pops up instead.

#### Steps to reproduce the behaviour
Float the mouse over the clock on the Panel, note the behaviour: the pop up with text "Click to view month calendar"

#### MATE general version
1.26
1.24

#### Package version
1.26
1.24

#### Linux Distribution
Ubuntu 21.10
Ubuntu 20.04 LTS

#### Link to bugreport of your Distribution (requirement)

There are moments when you need to keep tabs with time in different time zones. It could be your employer, relatives or friends live abroad or maybe you are tracking a sports event or a happening abroad. If that is the case, nothing can be more helpful than having multiple clocks for different time zones.

**Comment by lukefromdc** [comment]
Changing this could have the effect of entirely concealing the calendar from new users, I don't see that happening unless
the tooltip were expanded to include BOTH sets of information.

**Comment by HectorB-2020** [comment]
In my view clicking on clock became an essential and obvious step nowadays for everybody.

Who are these potential new users? The ones who are coming from Windows are already accustomed to revealing the calendar and agenda by clicking on the clock.

**Comment by lukefromdc** [comment]
Point I am making is that REMOVING the tooltip text about raising the calendar window would make that 
non-discoverable except by accident or reading the docs (which so many users do not do). That does not 
block adding text, only removing it.

**Comment by zhuyaliang** [comment]
You can add location information and click the system clock to view the time in different areas

**Comment by HectorB-2020** [comment]
Right, I certainly can do this. Showing multiple time zones this as a pop-up message merely improves user experience.
GUI is all about UX.
